src/otaclient/grpc/api_v2/servicer.py

```diff
@@ -111,6 +112,22 @@ async def update(
         update_acked_ecus = set()
         response = api_types.UpdateResponse()
 
+        # NOTE(20241220): due to the fact that OTA Service API doesn't have field
+        #                 in UpdateResponseEcu msg, the only way to pass the failure_msg
+        #                 to upper is by status API.
+        if not otaclient_cfg.ECU_INFO_LOADED_SUCCESSFULLY:
```

Comment:
Just comment.
Seems current otaclient just validate file existence and yaml format.
if otaclient expects specific keys, we can validate schema too in future PR.